### PR TITLE
John conroy/cells form labels

### LINKED ISSUE #2777: Update text for 'Genomic Modality' section in gene queries to match mockup
Screen Shot 2022-07-26 at 10 34 38 AM
This is an updated mockup that changes "Measurement" from original mockup to "Genomic Modality", which is the same as the current cells UI (make sure to fix typo for 'modality). If the assistive text ("Query measurement refers to....") takes too much extra work at the moment, then no need to do right now. Suggestive text should be changed to above screenshot.

The dropdown for that section also has 'rna' and 'atac', but if possible, change to below screenshot. 
Screen Shot 2022-07-26 at 10 40 49 AM

### PR DESCRIPTION
**Pull request by john-conroy**
Fix #2777.

Screen Shot 2022-08-04 at 2 32 48 PM

Screen Shot 2022-08-04 at 2 33 03 PM

Screen Shot 2022-08-04 at 2 33 14 PM

### CODE REVIEW

File: context/app/static/js/components/cells/DatasetsSelectedByExpression/DatasetsSelectedByExpression.jsx [hunk 1]
```diff
@@ -140,35 +122,41 @@ function DatasetsSelectedByExpression({
...
+          minLog={-4}
+          maxLog={5}
+          onChange={(e, val) => setMinExpressionLog(val)}
+          id="min-expression"
+        />
+      </div>
+      <div>
+        <MarkedSlider
+          label="Minimum Cell Percentage (%)"
+          helperText="Set the minimum cell percentage for cells in the datasets to represent the minimum expression level."
```
Comment (mccalluc) [inline context/app/static/js/components/cells/DatasetsSelectedByExpression/DatasetsSelectedByExpression.jsx:145]:
```suggestion
          helperText="Set the minimum cell percentage that must be above the minimum expression level for the dataset to be returned."
```

I have trouble understanding the current text... but I could also be confused about what this actually does.

**Review by mccalluc — APPROVED** [review]
Qualms about some of the text, but I'll defer to Tiffany's judgement. Looks good.

File: context/app/static/js/components/cells/AutocompleteEntity/AutocompleteEntity.jsx [hunk 2]
```diff
@@ -55,9 +62,11 @@ function AutocompleteEntity(props) {
         setSelected(value);
         setter(value.map((match) => match.full));
       }}
-      renderInput={(params) => (
+      renderInput={({ InputLabelProps, ...params }) => (
```
[the remark this answers is not in the document]
Comment (john-conroy) [reply context/app/static/js/components/cells/AutocompleteEntity/AutocompleteEntity.jsx:65]: This is a key in an object defined by MUI that is being destructured and is case sensitive.

File: context/app/static/js/components/cells/AutocompleteEntity/AutocompleteEntity.jsx [hunk 3]
```diff
@@ -5,9 +5,16 @@ import TextField from '@material-ui/core/TextField';
 
 import CellsService from 'js/components/cells/CellsService';
 
-function AutocompleteEntity(props) {
-  const { targetEntity, setter, setCellVariableNames } = props;
+function buildHelperText(entity) {
+  return `Multiple ${entity} are allowed and currently only 'AND' queries are supported.`;
```
[the remark this answers is not in the document]
Comment (tsliaw) [reply context/app/static/js/components/cells/AutocompleteEntity/AutocompleteEntity.jsx:9]: I would be fine with the changes, but can you only strike the 'currently' part so it says '...and only 'AND' queries...'

File: context/app/static/js/components/cells/DatasetsSelectedByExpression/DatasetsSelectedByExpression.jsx (hunk 1 above)
Comment (tsliaw) [reply context/app/static/js/components/cells/DatasetsSelectedByExpression/DatasetsSelectedByExpression.jsx:145]: Let's leave the original text for this one until there's user input on this.

**Comment by tsliaw** [comment]
Everything looks good except for the 'Genomic Modality' section. Can you fix the typo from 'Genomic Modaility' to 'Genomic Modality'? (there's an extra 'i' in the current text) Also in that section, instead of "Query measurement refers to...", can you change the helper text to "Genomic modality refers to..."

Otherwise, everything else looks good.

**Review by tsliaw — APPROVED** [review]

**Pull request merged**